v3/src/components/data-display/models/data-configuration-model.ts

```diff
@@ -58,9 +58,23 @@ export const DataConfigurationModel = types
   .volatile(() => ({
     actionHandlerDisposer: undefined as (() => void) | undefined,
     filteredCases: undefined as FilteredCases[] | undefined,
-    handlers: new Map<string, (actionCall: ISerializedActionCall) => void>(),
-    pointsNeedUpdating: false
+    handlers: new Map<string, (actionCall: ISerializedActionCall) => void>()
   }))
+  .extend(self => {
+    let pointsNeedUpdating = false
+
+    return {
+      views: {
+        get pointsNeedUpdating() {
+          return pointsNeedUpdating
+        },
```

Comment:
I think I could go back to using volatile at this point. I moved it early on because there was a lot of noise during application launch that I was trying to make sense of, but you're right since no patches are generated it shouldn't affect the undo history.